app/models/gogovan_order.rb

```diff
@@ -37,6 +39,33 @@ def cancel_order
     update_status('cancelled')
   end
 
+  def push_back_offer_state
+    delivery.offer.cancel_schedule
+  end
+
+  def assign_details(order_details)
+    self.status = order_details["status"]
+    self.price = order_details["price"]
+    if (driver_details = order_details["driver"])
+      self.driver_name = driver_details["name"]
+      self.driver_mobile = driver_details["phone_number"]
+      self.driver_license = driver_details["license_plate"]
+    end
+    self
+  end
+
+  private
+
+  def start_polling_status
+    PollGogovanOrderStatusJob.set(wait: GGV_POLL_JOB_WAIT_TIME).perform_later(self.id)
+  end
+
+  def self.set_vehicle_type(attributes)
+    offer = Offer.find(attributes['offerId'])
+    attributes['vehicle'] = offer.gogovan_transport.vehical_tag
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.